.rhcicd/clowdapp-connector-email.yaml

```
@@ -0,0 +1,231 @@
+        - name: QUARKUS_LOG_SENTRY_ENVIRONMENT
+          value: ${ENV_NAME}
+        - name: QUARKUS_REST_CLIENT_IT_S2S_KEY_STORE
+          value: ${IT_SERVICE_TO_SERVICE_KEY_STORE}
+        - name: QUARKUS_REST_CLIENT_IT_S2S_KEY_STORE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: it-services
+              key: it-services-keystorepassword
+              optional: true
+        - name: QUARKUS_REST_CLIENT_IT_S2S_URL
+          valueFrom:
+            secretKeyRef:
+              name: it-services
+              key: url
+              optional: true
+parameters:
+- name: CPU_LIMIT
+  description: CPU limit
+  value: 200m
+- name: CPU_REQUEST
+  description: CPU request
+  value: 100m
+
+- name: ENV_NAME
+  description: ClowdEnvironment name (ephemeral, stage, prod)
+  required: true
+
+- name: IMAGE
+  description: Image URL
+  value: quay.io/cloudservices/notifications-connector-email
+- name: IMAGE_TAG
+  value: latest
+- name: IT_SERVICE_TO_SERVICE_KEY_STORE
+  description: "Key store for opening a secure connection when communicating with IT. It should be set to 'file:/mnt/secrets/clientkeystore.jks'"
+  value: ""
+
+
+- name: KAFKA_CLIENT_LOG_LEVEL
+  description: Log level of the Kafka client library
+  value: WARN
+- name: KAFKA_MAX_POLL_INTERVAL_MS
+  description: Maximum delay in milliseconds between two calls of poll(). Defaults to 300000 (5 minutes).
+  value: "300000"
+- name: KAFKA_MAX_POLL_RECORDS
+  description: Maximum number of records returned in a single call of poll(). Defaults to 500.
+  value: "500"
+- name: KAFKA_POLL_ON_ERROR
+  description: What to do if kafka threw an exception while polling for new messages. See https://camel.apache.org/components/latest/kafka-component.html for more details.
+  value: RECONNECT
+
+- name: MEMORY_LIMIT
+  description: Memory limit
+  value: 500Mi
+- name: MEMORY_REQUEST
+  description: Memory request
+  value: 250Mi
+- name: MIN_REPLICAS
+  value: "3"
+
+- name: NOTIFICATIONS_CONNECTOR_ENDPOINT_CACHE_MAX_SIZE
+  description: Maximum size of the Camel endpoints cache
+  value: "100"
+- name: NOTIFICATIONS_CONNECTOR_FETCH_USERS_RBAC_ENABLED
+  description: Is RBAC used to fetch the users?
+  value: "true"
+- name: NOTIFICATIONS_CONNECTOR_HTTPS_CONNECT_TIMEOUT_MS
+  description: Maximum time in milliseconds allowed to establish an HTTPS connection
+  value: "2500"
+- name: NOTIFICATIONS_CONNECTOR_HTTPS_SOCKET_TIMEOUT_MS
+  description: Maximum time in milliseconds allowed to wait for HTTPS data
+  value: "2500"
+- name: NOTIFICATIONS_CONNECTOR_REDELIVERY_DELAY
+  description: Delay in milliseconds between two redelivery attempts
+  value: "1000"
+- name: NOTIFICATIONS_CONNECTOR_REDELIVERY_MAX_ATTEMPTS
+  description: Maximum number of redelivery attempts (initial call not included)
+  value: "2"
+- name: NOTIFICATIONS_CONNECTOR_SINGLE_EMAIL_PER_USER_ENABLED
+  description: Is sending a single email per user enabled?
+  value: "true"
```

Let's start with the same configuration than prod.
```suggestion
  value: "false"
```

---

Changed!